src/index.css

```diff
@@ -51,7 +51,7 @@ img {
   max-width: 300px;
   border: 1px solid #ccc;
   border-radius: 4px;
-} */
+} 
 
 img.logo {
   border: none;
```

Comment:
The design is simple but neat. I think this is better than overcomplicating. More explanatory text could be nice. I know you wanted to spend more time on design than you got to